doc/api/async_hooks.md

```diff
@@ -29,7 +29,7 @@ Following is a simple overview of the public API.
 const async_hooks = require('async_hooks');
 
 // Return the ID of the current execution context.
-const cid = async_hooks.currentId();
+const cid = async_hooks.currentAsyncId();
```

Comment:
Something that confuses me about the `currentId` renaming is that `triggerId` refers to an `asyncId`. So in my mind `currentTriggerAsyncId()` is what makes sense. Of cause that is a horribly long name and it doesn't change that `currentAsyncId()` isn't really specific. It just says it is an `asyncId` that is current, but the same could be said about `triggerId`.

My previous understanding was that `current` had some special meaning, and a logical renaming of `triggerId` would then be  `triggerAsyncId()`. But this comment invalidates that understanding.